src/gennylib/src/PoolFactory.cpp

```diff
@@ -72,6 +72,9 @@ struct PoolFactory::Config {
             queryOptions[key] = value;
             i += matches.length();
         }
+
+        // Hardcode Genny appName
+        queryOptions["appName"] = "Genny";
```

Comment:
Please add a comment here saying the appName is not strictly a query option and the `queryOptions` can be renamed if we intend to continue to use for generic key value pairs in the URI.